clients/privacy-center/package.json

```diff
@@ -35,6 +35,7 @@
     "fidesui": "*",
     "formik": "^2.2.9",
     "framer-motion": "^11.2.12",
+    "git-tag-version": "^1.3.1",
```

Comment:
I think this would make more sense to do in the `clients/fides-js` project instead, as part of the rollup build there

Reply:
I'll try that